packages/epo-react-lib/package.json

```diff
@@ -23,6 +23,7 @@
     "@babel/preset-env": "^7.19.1",
     "@babel/preset-react": "^7.18.6",
     "@babel/preset-typescript": "^7.18.6",
+    "@castiron/style-mixins": "^1.0.6",
```

Comment:
When I ran `yarn` from the root of the repo it updated the `yarn.lock` and I got a diff.  And as I don't see and `yarn.lock` changes in this PR, my guess is that change to `yarn.lock` ought be checked in too.